benchmarks/driver.cpp

```diff
@@ -163,6 +163,79 @@ static ErrorObject benchmarkConvFprop(
   return ok();
 }
 
+static ErrorObject benchmarkMatmul(int64_t m, int64_t n, int64_t k, bool transA,
+                                   bool transB, int64_t iter, bool dump,
+                                   DataType matmulIOType, int64_t deviceId) {
+#ifdef FUSILLI_ENABLE_AMDGPU
+  Handle handle = FUSILLI_TRY(Handle::create(Backend::AMDGPU, deviceId));
+#else
+  Handle handle = FUSILLI_TRY(Handle::create(Backend::CPU));
+#endif
+
+  // Build attributes based on transpose flags.
+  auto aDims = std::vector<int64_t>{m, k};
+  auto bDims = std::vector<int64_t>{k, n};
+  auto aStride =
+      transA ? std::vector<int64_t>{1, m} : std::vector<int64_t>{k, 1};
+  auto bStride =
+      transB ? std::vector<int64_t>{1, k} : std::vector<int64_t>{n, 1};
+
+  Graph graph;
+  auto graphName = std::format("benchmark_matmul_m{}_n{}_k{}_transA{}_transB{}",
```

Comment:
Can you add dtype to the name here to disambiguate dumps of the same mnk/trans but diff dtypes?